vit-servicing-station-lib/src/v0/mod.rs

```diff
@@ -16,9 +16,33 @@ pub async fn filter(
     let v0_root = warp::path!("v0" / ..);
     let service_version_root = warp::path!("vit-version" / ..);
     // log request statistics
-    let log = warp::log::custom(|info| {
-        tracing::info!("Request elapsed time: {}ms", info.elapsed().as_millis())
+    let log = warp::filters::trace::trace(|info| {
+        use http_zipkin::get_trace_context;
+        use tracing::field::Empty;
+        let span = tracing::span!(
+            tracing::Level::DEBUG,
+            "rest_api_request",
+            method = %info.method(),
+            path = info.path(),
+            version = ?info.version(),
+            remote_addr = Empty,
+            trace_id = Empty,
+            span_id = Empty,
+            parent_span_id = Empty,
+        );
+        if let Some(remote_addr) = info.remote_addr() {
+            span.record("remote_addr", &remote_addr.to_string().as_str());
```

Comment:
Is this going to be the address of the load balancer in our infrastructure?

Reply:
I think is the remote address making the request. It is saved in the `remote_addr` record attribute

Reply:
I'm not sure this is what warp puts into the `info` structure. Or does it parse the headers set up by the proxy to get at the original source IP address?